spec/line/bot/client_get_follower_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+require 'webmock/rspec'
+require 'json'
+
+USER_ID_CONTENT = <<"EOS"
+{
+    "userIds": [
+        "Uxxxxxxxxxxxxxx1",
+        "Uxxxxxxxxxxxxxx2",
+        "Uxxxxxxxxxxxxxx3"
+    ],
+    "next": "jxEWCEEP"
+}
+EOS
+
+NEXT_USER_ID_CONTENT = <<"EOS"
+{
+    "userIds": [
+        "Uxxxxxxxxxxxxxx4",
+        "Uxxxxxxxxxxxxxx5",
+        "Uxxxxxxxxxxxxxx6"
+    ]
+}
+EOS
+
+describe Line::Bot::Client do
+  before do
+  end
```

Comment:
I referred to other specs, do we need this block? 🤔

Reply:
I think it is no need.

Reply:
Removed this block.